src/App.js

```diff
@@ -9,7 +10,9 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    <Article title={article.title} text={article.text} date={article.date} />
```

Comment:
```suggestion
    <Article 
      title={article.title}
      text={article.text}
      date={article.date}
    />
```
Компоненты с >1 props записывай в таком виде